src/v_video.c

```diff
@@ -63,6 +64,12 @@ int dirtybox[4];
 // This is needed for Chocolate Strife, which clips patches to the screen.
 static vpatchclipfunc_t patchclip_callback = NULL;
 
+// We use a different scaler for PNG screenshots from screen rendering.
+// Default value is the smallest required for a lossless aspect-correct shot.
+static screen_mode_t * screenshot_scaler;
```

Comment:
Please remove spaces after *, declare pointer variables like this:

    static screen_mode_t *screenshot_scaler;